YoutubeDownloader/Views/Dialogs/SettingsView.xaml

```diff
@@ -168,6 +169,26 @@
                                 Value="{Binding ParallelLimit}" />
                         </StackPanel>
                     </DockPanel>
+                    
+                    <Button
+                        Margin="16"
+                        HorizontalAlignment="Stretch"
+                        Command="{s:Action Logout}"
+                        Content="Logout"
+                        IsCancel="True"
+                        IsDefault="True"
+                        Visibility="{Binding IsLogged, Converter={x:Static converters:BoolToVisibilityConverter.VisibleOrCollapsed}}"
+                        Style="{DynamicResource MaterialDesignOutlinedButton}" />
```

Comment:
Do you think we can display the username here? So that the user knows what account they're logged in as

Reply:
Making a request to `https://www.youtube.com/youtubei/v1/account/account_menu?key=AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8&prettyPrint=false` returns `accountName` and `channelHandle` too, and I don't know if there is any other way to get it.
Do we do it this way?

Reply:
If we need to make a request, then it's not worth it. I was hoping there would be something available directly after logging in 🤔